### ISSUE TITLE
NPE while opening Chunky

### ISSUE THREAD

**Issue by ThatRedox** [leMaik/chunky-launcher-standalone#7]
Null pointer exception while loading. May or may not cause other issues, Chunky appears to work fine.
```
Exception in thread "JavaFX Application Thread" java.lang.NullPointerException: Cannot invoke "se.llbit.chunky.renderer.export.PictureExportFormat.getName()" because "object" is null
	at se.llbit.chunky.ui.render.AdvancedTab$1.toString(AdvancedTab.java:114)
	at se.llbit.chunky.ui.render.AdvancedTab$1.toString(AdvancedTab.java:111)
	at javafx.controls/javafx.scene.control.skin.ChoiceBoxSkin.getDisplayText(Unknown Source)
	at javafx.controls/javafx.scene.control.skin.ChoiceBoxSkin.updateLabelText(Unknown Source)
	at javafx.controls/javafx.scene.control.skin.ChoiceBoxSkin.initialize(Unknown Source)
	at javafx.controls/javafx.scene.control.skin.ChoiceBoxSkin.<init>(Unknown Source)
	at javafx.controls/javafx.scene.control.ChoiceBox.createDefaultSkin(Unknown Source)
	at javafx.controls/javafx.scene.control.Control.doProcessCSS(Unknown Source)
	at javafx.controls/javafx.scene.control.Control$1.doProcessCSS(Unknown Source)
	at javafx.controls/com.sun.javafx.scene.control.ControlHelper.processCSSImpl(Unknown Source)
	at javafx.graphics/com.sun.javafx.scene.NodeHelper.processCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Parent.doProcessCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Parent$1.doProcessCSS(Unknown Source)
	at javafx.graphics/com.sun.javafx.scene.ParentHelper.processCSSImpl(Unknown Source)
	at javafx.graphics/com.sun.javafx.scene.NodeHelper.processCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Parent.doProcessCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Parent$1.doProcessCSS(Unknown Source)
	at javafx.graphics/com.sun.javafx.scene.ParentHelper.processCSSImpl(Unknown Source)
	at javafx.graphics/com.sun.javafx.scene.NodeHelper.processCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Parent.doProcessCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Parent$1.doProcessCSS(Unknown Source)
	at javafx.graphics/com.sun.javafx.scene.ParentHelper.processCSSImpl(Unknown Source)
	at javafx.graphics/com.sun.javafx.scene.NodeHelper.processCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Parent.doProcessCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Parent$1.doProcessCSS(Unknown Source)
	at javafx.graphics/com.sun.javafx.scene.ParentHelper.processCSSImpl(Unknown Source)
	at javafx.graphics/com.sun.javafx.scene.NodeHelper.processCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Parent.doProcessCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Parent$1.doProcessCSS(Unknown Source)
	at javafx.graphics/com.sun.javafx.scene.ParentHelper.processCSSImpl(Unknown Source)
	at javafx.controls/com.sun.javafx.scene.control.ControlHelper.superProcessCSSImpl(Unknown Source)
	at javafx.controls/com.sun.javafx.scene.control.ControlHelper.superProcessCSS(Unknown Source)
	at javafx.controls/javafx.scene.control.Control.doProcessCSS(Unknown Source)
	at javafx.controls/javafx.scene.control.Control$1.doProcessCSS(Unknown Source)
	at javafx.controls/com.sun.javafx.scene.control.ControlHelper.processCSSImpl(Unknown Source)
	at javafx.graphics/com.sun.javafx.scene.NodeHelper.processCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Parent.doProcessCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Parent$1.doProcessCSS(Unknown Source)
	at javafx.graphics/com.sun.javafx.scene.ParentHelper.processCSSImpl(Unknown Source)
	at javafx.graphics/com.sun.javafx.scene.NodeHelper.processCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Node.processCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Node.processCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Node.processCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Node.processCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Node.processCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Node.processCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Node.processCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Node.processCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Node.processCSS(Unknown Source)
	at javafx.graphics/javafx.scene.Scene.doCSSPass(Unknown Source)
	at javafx.graphics/javafx.scene.Scene$ScenePulseListener.pulse(Unknown Source)
	at javafx.graphics/com.sun.javafx.tk.Toolkit.lambda$runPulse$2(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at javafx.graphics/com.sun.javafx.tk.Toolkit.runPulse(Unknown Source)
	at javafx.graphics/com.sun.javafx.tk.Toolkit.firePulse(Unknown Source)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.pulse(Unknown Source)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.pulse(Unknown Source)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.pulseFromQueue(Unknown Source)
	at javafx.graphics/com.sun.javafx.tk.quantum.QuantumToolkit.lambda$runToolkit$11(Unknown Source)
	at javafx.graphics/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(Unknown Source)
	at javafx.graphics/com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at javafx.graphics/com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
```

**Comment by ThatRedox** [comment]
https://github.com/chunky-dev/chunky/issues/1016

**Comment by leMaik** [comment]
Can confirm this was fixed by chunky-dev/chunky#1016
